public/resources/img/icon/touch-emulator-hover.svg

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Capa_1" x="0px" y="0px" width="512px" height="512px" viewBox="0 0 569.867 569.867" style="enable-background:new 0 0 569.867 569.867;" xml:space="preserve">
+<g>
+	<path d="M414.147,0H155.71c-21.238,0-38.422,17.212-38.422,38.441v492.976c0,21.238,17.184,38.45,38.422,38.45h258.427   c21.229,0,38.441-17.212,38.441-38.45V38.432C452.579,17.203,435.366,0,414.147,0z M284.288,546.382   c-13.378,0-24.202-10.815-24.202-24.184c0-13.358,10.824-24.184,24.202-24.184c13.33,0,24.184,10.825,24.184,24.184   C308.481,535.566,297.618,546.382,284.288,546.382z M444.871,493.96c-1.893,7.229-6.855,13.302-13.607,16.706l-52.039,26.46   c-3.528,1.854-7.669,2.888-11.78,2.888c-9.792,0-18.638-5.585-23.648-14.937c-2.046-3.863-5.996-9.065-13.492-12.365   c-25.131-11.063-42.831-29.031-55.443-43.451c-1.358-1.55-2.945-3.28-4.695-5.154h-90.433c-5.842,0-10.576-4.733-10.576-10.576   V73.995c0-5.843,4.734-10.567,10.576-10.567h210.375c5.843,0,10.585,4.724,10.585,10.567v292.517l25.608,28.267   c6.704,7.411,11.37,19.526,11.37,29.472v38.259c0,2.362,1.626,7.66,2.945,9.61C446.535,480.869,446.201,488.825,444.871,493.96z" fill="#006DF0"/>
+	<path d="M425.125,465.99v-38.26c0-6.378-3.366-15.09-7.659-19.842l-30.601-33.766c-1.511-1.683-3.223-3.155-5.097-4.36   c-3.528-2.267-7.229-2.448-8.281-2.448l-2.543,0.354c-0.526,0.183-0.985,0.383-1.396,0.612l-10.901-43.185   c-1.692-6.722-7.717-12.01-13.693-12.01c-6.034,0-11.504,5.661-12.202,12.612l-10.299,103.619   c-0.191,2.143-1.014,5.977-4.093,7.449c-2.237,1.1-5.326,0.526-8.386-1.434l-30.543-19.709c-5.757-3.682-14.563-2.658-19.269,2.075   c-2.419,2.419-3.758,5.623-3.729,9.056c0.038,3.442,1.463,6.655,3.978,9.037c11.819,11.303,21.181,20.808,27.033,27.482   c11.638,13.301,27.884,29.835,50.413,39.761c11.274,4.953,17.299,12.909,20.368,18.724c2.468,4.6,6.561,7.372,10.997,7.372   c1.816,0,3.614-0.449,5.288-1.329l52.077-26.479c3.089-1.559,5.346-4.284,6.178-7.487c0.832-3.223,0.2-6.589-2.237-10.203   C427.611,479.33,425.125,471.24,425.125,465.99z" fill="#006DF0"/>
+	<path d="M319.029,322.476c0.421,0.115,0.832,0.173,1.243,0.173c2.113,0,4.045-1.406,4.608-3.538   c0.039-0.134,3.663-12.957,19.489-13.599c15.769-0.525,20.253,10.663,20.425,11.103c0.928,2.477,3.701,3.758,6.149,2.821   c2.485-0.928,3.738-3.673,2.82-6.149c-0.277-0.745-6.809-18.178-29.777-17.337c-18.207,0.746-26.488,13.847-28.324,20.675   C314.964,319.177,316.485,321.797,319.029,322.476z" fill="#006DF0"/>
+	<path d="M258.211,410.374c12.91-8.778,22.845-1.635,23.218-1.367c2.084,1.616,5.087,1.253,6.703-0.86   c1.616-2.095,1.233-5.097-0.86-6.703c-0.622-0.497-15.635-11.762-34.435,1.022c-15.08,10.252-15.185,25.752-13.129,32.522   c0.622,2.056,2.515,3.366,4.562,3.366c0.449,0,0.918-0.067,1.358-0.201c2.524-0.736,3.958-3.395,3.242-5.919   C248.715,431.718,245.292,419.162,258.211,410.374z" fill="#006DF0"/>
+</g>
+<g>
```

Comment:
Please run this SVG through [svgo](https://www.npmjs.com/package/svgo) so that all the "bits that do nothing", comments, nonstandard attributes, etc. get removed